### PR TITLE
Update prerequisites.rst

### PR DESCRIPTION
**Pull request by tulhaum**
Adds "RewriteBase /glpi" to complement the Alias directive when placing GLPI in a subfolder.

### CODE REVIEW

**Review by trasher — COMMENTED** [review]
Maybe should it precise the value for the directive should be the same as the used alias? (if I'm not wrong)

File: source/prerequisites.rst [hunk 1]
```diff
@@ -45,6 +45,8 @@ Here is a virtual host configuration example for ``Apache 2`` web server.
             # Redirect all requests to GLPI router, unless file exists.
             RewriteCond %{REQUEST_FILENAME} !-f
             RewriteRule ^(.*)$ index.php [QSA,L]
+            # If you are going to use 'Alias' directive shown above, you need set RewriteBase:
+            # RewriteBase /glpi
```
Comment (cedric-anne) [inline source/prerequisites.rst:49]: I did not had to use it when I made tests on this part. Are your rewrite directives placed inside the vhost configration file, or inside a `.htaccess` file ?

Comment (tulhaum) [reply source/prerequisites.rst:49]: My rewrite directives are placed inside vhost configuration. I've tried several ways, this is the only one I've had success with.

**Comment by tulhaum** [comment]
> Maybe should it precise the value for the directive should be the same as the used alias? (if I'm not wrong)

Yes, in my case it showed that.

**Review by cedric-anne — COMMENTED** [review]
I made some tests and this directive has no effect, unless you use the same path for both `DocumentRoot` and `Alias` directive, which is a mistake (see #88).

If you want to serve GLPI in a subdirectory of your webserver, you should not place the GLPI source code in the document root directory, but in another directory.

For instance, you can keep the default `DocumentRoot /var/www/html` and place GLPI in `/var/www/glpi`.

```
<VirtualHost *:80>
    DocumentRoot /var/www/html       

    Alias "/glpi" "/var/www/glpi/public"

    <Directory /var/www/glpi/public>
        Require all granted

        RewriteEngine On

        # Redirect all requests to GLPI router, unless file exists.
        RewriteCond %{REQUEST_FILENAME} !-f
        RewriteRule ^(.*)$ index.php [QSA,L]
    </Directory>
</VirtualHost>
```

**Pull request closed without merging**